playground/ssr-vue/__tests__/ssr-vue.spec.ts

```diff
@@ -159,7 +159,9 @@ test('hydration', async () => {
 })
 
 test('hmr', async () => {
-  await page.goto(url)
+  // This is test is flaky in Mac CI, but can't be reproduced locally. Wait until
+  // network idle to avoid the issue
```

Comment:
```suggestion
  // network idle to avoid the issue. TODO: This may be caused by a bug when
  // modifying a file while loading, we should remove this guard
```